### PR TITLE
Move libnetwork-plugin to a standalone Docker image

### PR DESCRIPTION
**Pull request by tomdee**

### CODE REVIEW

**Comment by tomdee** [comment]
@robbrockbank Can you review?

File: Dockerfile [hunk 1]
```diff
@@ -11,11 +11,19 @@
...
 
-COPY node_filesystem /
+COPY requirements.txt /
+
+RUN apk --update add python py-setuptools && \
+    apk add --virtual build-dependencies git python-dev build-base curl bash py-pip alpine-sdk libffi-dev openssl-dev && \
+    pip install -r requirements.txt && \
+    apk del build-dependencies && rm -rf /var/cache/apk/*
+
+COPY start.sh /
```
Comment (robbrockbank) [inline Dockerfile]: I don't think the trailing / shouldn't be there

File: Makefile [hunk 2]
```diff
@@ -67,6 +65,10 @@ st: calicoctl busybox.tar calico-node.tar run-etcd run-consul
 fast-st: busybox.tar calico-node.tar run-etcd run-consul
 	nosetests $(ST_TO_RUN) -sv --nologcapture --with-timer -a '!slow'
 
+run-plugin:
+	docker run -ti --uts="host" -v /run/docker/plugins:/run/docker/plugins -e ETCD_AUTHORITY=$(LOCAL_IP_ENV):2379 calico/node-libnetwork
```
Comment (robbrockbank) [inline Makefile]: bind the log directory too?

File: libnetwork/docker_plugin.py [hunk 3]
```diff
@@ -64,7 +64,7 @@ def make_json_error(ex):
 
 app = make_json_app(__name__)
 app.logger.addHandler(logging.StreamHandler(sys.stdout))
-app.logger.setLevel(logging.INFO) # TODO - make this configuration dynamically
+app.logger.setLevel(logging.DEBUG) # TODO - make this configuration dynamically
```
Comment (robbrockbank) [inline libnetwork/docker_plugin.py]: I assume we'll set this back to INFO at some point?

**Comment by robbrockbank** [comment]
LGTM

**Pull request closed without merging**